### ISSUE TITLE
getBox error on 0.15.0

### ISSUE THREAD

**Issue by GantMan** [justadudewhohacks/face-api.js#120]
### Calling allFaces with SsdMobilenetv1 is giving me a getBox error.

This code:
```js
const faceImage = await faceapi.fetchImage('https://i.imgur.com/GzBMITw.jpg')
const faceDescript = await faceapi.allFacesSsdMobilenetv1(faceImage)
```
resulting in:
![image](https://user-images.githubusercontent.com/997157/47692693-a732d700-dbc4-11e8-9d81-588766fb61fd.png)

I have the code (in React) available upon request.  Please let me know if this issue makes sense or if I should do more.

**Comment by justadudewhohacks** [comment]
Apparently I forgot to clean the build folder before building and publishing, because the error originates from an old file that has been removed.

**Comment by justadudewhohacks** [comment]
Should be resolved in 0.15.1.

**Comment by GantMan** [comment]
it is
